license-classifications.yml

```
@@ -3464,23 +3470,23 @@ categorizations:
   # https://spdx.org/licenses/CC-BY-NC-SA-1.0.html
   - id: "CC-BY-NC-SA-1.0"
     categories:
-      - "free-restricted"
+      - "copyleft-strong"
       - "property:include-in-notice-file"
       - "property:non-commercial"
       - "property:creativecommons"
 
   # https://spdx.org/licenses/CC-BY-NC-SA-2.0.html
   - id: "CC-BY-NC-SA-2.0"
     categories:
-      - "free-restricted"
+      - "copyleft-strong"
```

```suggestion
      - "free-restricted"
```
As all CC-licenses with the NC-addition are not open source, we do not apply the catergory "copyleft-strong" to these. Insteast the category "free-restricted" would seem the most appropriate, as long as the "property:noncommercial" is also in place.